qa/rpc-tests/wallet.py

```diff
@@ -29,42 +30,43 @@ def setup_network(self, split=False):
         self.sync_all()
 
     def run_test (self):
-
+        immature_balance = 40.00000000
+        
         # Check that there's no UTXO on none of the nodes
         assert_equal(len(self.nodes[0].listunspent()), 0)
         assert_equal(len(self.nodes[1].listunspent()), 0)
         assert_equal(len(self.nodes[2].listunspent()), 0)
 
-        print("Mining blocks...")
-
+        # 40 zc
         self.nodes[0].generate(1)
 
         walletinfo = self.nodes[0].getwalletinfo()
-        assert_equal(walletinfo['immature_balance'], 50)
+        assert_equal(walletinfo['immature_balance'], immature_balance)
         assert_equal(walletinfo['balance'], 0)
 
         self.sync_all()
         self.nodes[1].generate(101)
         self.sync_all()
 
-        assert_equal(self.nodes[0].getbalance(), 50)
-        assert_equal(self.nodes[1].getbalance(), 50)
+        assert_equal(self.nodes[0].getbalance(), immature_balance)
+        assert_equal(self.nodes[1].getbalance(), immature_balance)
         assert_equal(self.nodes[2].getbalance(), 0)
 
         # Check that only first and second nodes have UTXOs
         assert_equal(len(self.nodes[0].listunspent()), 1)
         assert_equal(len(self.nodes[1].listunspent()), 1)
         assert_equal(len(self.nodes[2].listunspent()), 0)
 
-        # Send 21 BTC from 0 to 2 using sendtoaddress call.
-        self.nodes[0].sendtoaddress(self.nodes[2].getnewaddress(), 11)
+        # Send 21 Zcoin from 0 to 2 using sendtoaddress call.
         self.nodes[0].sendtoaddress(self.nodes[2].getnewaddress(), 10)
+        time.sleep(5)
```

Comment:
Why do we need to sleep?

Reply:
This test very unstable. Sometimes need to wait till transactions will be received by node. Need to investigate how to improve this test. I think now is a bad time to uncomment it in common regtests file.

Reply:
it should be solved after disable dandelion. isn't it?